AutomatedTesting/Gem/PythonTests/Atom/atom_utils/atom_constants.py

```diff
@@ -315,6 +315,7 @@ def decal(property: str = 'name') -> str:
             'name': 'Decal',
             'Attenuation Angle': 'Controller|Configuration|Attenuation Angle',
             'Opacity': 'Controller|Configuration|Opacity',
+            'Normal Map Opacity': 'Controller|Configuration|Normal Map Opacity',
```

Comment:
docstring for this new key is missing.
`  - 'Normal Map Opacity' somthing something normal map opacity description`

Reply:
added